File: app/src/main/java/com/example/jetpackcomponentsapp/view/IndicatorViewHolder.java
package com.example.jetpackcomponentsapp.view;

import android.content.Context;
import android.util.Log;
import android.view.View;

import androidx.appcompat.widget.AppCompatImageView;

import com.example.jetpackcomponentsapp.R;

public class IndicatorViewHolder extends BaseViewHolder {

    static private final String TAG = IndicatorViewHolder.class.getSimpleName();
    //region cell_circle
    private final AppCompatImageView image;
    //endregion
    public IndicatorViewHolder(Context context, View itemView) {
        super(context, itemView);
        Log.d(TAG, "constructor");
        image = itemView.findViewById(R.id.image);
    }

    public void bindDataToViewHolder() {

    }
}

File: app/src/main/java/com/example/jetpackcomponentsapp/view/IndicatorViewHolder.kt
package com.example.jetpackcomponentsapp.view

import android.util.Log
import android.view.View
import androidx.appcompat.widget.AppCompatImageView
import com.example.jetpackcomponentsapp.R

public class IndicatorViewHolder : BaseViewHolder {

    companion object {
        private val TAG = IndicatorViewHolder::class.java.getSimpleName()
    }
    //region cell_circle
    private val image : AppCompatImageView
    //endregion
    constructor(itemView : View) : super(itemView) { Log.d(TAG, "constructor")
        image = itemView.findViewById(R.id.image)
    }

    public fun bindDataToViewHolder() {

    }
}